packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_ServiceOutput.ts

```diff
@@ -27,11 +28,15 @@ export abstract class ServiceOutput implements Hashable {
 }
 
 export class GraphFetchServiceOutput extends ServiceOutput {
+  keys!: object[];
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.